recipes/gitlab-runner/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set import_path = "gitlab.com/gitlab-org/gitlab-runner" %}
+{% set version = "12.1.0" %}
+{% set branch = '-'.join(version.split('.')[:2] +['stable']) %}
+
+{% set name = import_path.split('/')[-1] %}
+{% set pkg_src = ('src/'+import_path).replace("/", os.sep) %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  - folder: {{ pkg_src }}
+    url: https://{{ import_path }}/-/archive/v{{ version }}/{{ name }}-v{{ version }}.tar.bz2
+    sha256: 49e7993e1eabc362ec99562e84931e699b77948efb6669f3e98a5dc40f2b2b66
+
+build:
+  number: 0
+  script:
+    - pushd {{ pkg_src }}
+    - go install -v
+        -ldflags "-X {{import_path}}/common.NAME='{{ name }}'
+                  -X {{import_path}}/common.VERSION='{{ version }}'
+                  -X {{import_path}}/common.REVISION='deadbeef' 
+                  -X {{import_path}}/common.BRANCH='{{ branch }}'
+                  -X {{import_path}}/common.BUILT='2019-08-01T00:00:00z'"
```

```suggestion
                  -X {{ import_path }}/common.BUILT='2019-08-01T00:00:00z'"
```

Also, is there a way to get `conda build` to automatically insert the build date for us?

---

Sorry about that. I had not noticed you had the build-date logic for windows and linux in your original commit. I added it back into the recipe.

---

The date issue is fixed now as well. 

This recipe is ready for review/merge.